### ISSUE TITLE
Gitlab private token cannot be remove

### ISSUE THREAD

**Issue by pluskal** [LibreCodeCoop/GitLabBundle#3]
In GitLab configuration, `private token` cannot be removed. This way, the plugin cannot be deactivated.

**Comment by mrschaal** [comment]
@pluskal is this still an issue? We're considering trying this plugin out.

**Comment by pluskal** [comment]
Yes, to best of my knowledge. We have this plugin and I needed to create an dedicated user for it.
